client/boot/common.js

```diff
@@ -47,9 +47,15 @@ import { setupLocale } from './locale';
 const debug = debugFactory( 'calypso' );
 
 const setupContextMiddleware = ( reduxStore, reactQueryClient ) => {
+	let previousPath = null;
+
 	page( '*', ( context, next ) => {
 		const parsed = getUrlParts( context.canonicalPath );
 		const path = parsed.pathname + parsed.search || null;
+
+		context.previousPath = previousPath;
+		previousPath = context.path;
```

Comment:
Adding this prop so we're able to get the previous page. The existing `prevPath` prop doesn't do what the name suggests, always staying `false`.

Reply:
Can we fix it instead? I think having duplicate `prevPath` and `previousPath` is likely to cause confusion for other developers down the line.

Reply:
That's true. Thankfully, there are not many use cases for `prevPath`, I'll have a dig.

Reply:
`prevPath` is now gone.